app_dart/lib/src/request_handlers/update_existing_flaky_issues.dart

```diff
@@ -143,8 +145,14 @@ class UpdateExistingFlakyIssue extends ApiRequestHandler<Body> {
           builderFlakyMap[statistic.name] == true &&
           // ignore: iterable_contains_unrelated_type
           !ignoreFlakyMap.containsKey(statistic.name)) {
-        await _addCommentToExistingIssue(gitHub, slug,
-            bucket: Bucket.staging, statistic: statistic, existingIssue: nameToExistingIssue[statistic.name]!);
+        await _addCommentToExistingIssue(
+          gitHub,
+          slug,
+          bucket: Bucket.staging,
+          statistic: statistic,
+          existingIssue: nameToExistingIssue[statistic.name]!,
+          ciYaml: ciYaml,
+        );
```

Comment:
Can we update the tests as well? It's easier to use the `generateCiYaml` instead of hard coding a raw ci.yaml file in the test

Reply:
Yeah I can add a new method for generateCiYaml.

Reply:
I just reread what you wrote and there is no generateCiYaml function in the code currently.

Reply:
See https://cs.opensource.google/flutter/cocoon/+/main:app_dart/test/request_handlers/check_flaky_builders_test_data.dart which hard codes a file instead of using the proto or one of the example fakes from `FakeScheduler`

Reply:
Isn't returning the file contents the intent of the test?

Reply:
I have confirmed that this is required since the UpdateExistingFlakyIssue endpoint, when called, immediately goes to github to get the raw ciyaml file contents. See 
https://github.com/flutter/cocoon/blob/main/app_dart/lib/src/request_handlers/update_existing_flaky_issues.dart#L41 now if you still don't want the text I can add toString and Json methods.

Reply:
You can inject the ci.yaml.

`@visibleForTesting this.ciYaml`

If it's null, it can be pulled from GitHub.